src/main/Database/Form/FormDao.java

```diff
@@ -0,0 +1,19 @@
+package Database.Form;
+
+import Database.Dao;
+import Form.Form;
+import org.bson.types.ObjectId;
+
+import java.util.Optional;
+
+public interface FormDao extends Dao<Form> {
+  Optional<Form> get(ObjectId id);
+
+  Optional<Form> getByFileId(ObjectId fileId);
+
+  Optional<Form> get(String username);
```

Comment:
there will be multiple forms for a given user. I think you should return a Set<Form> instead and update the rest